ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -835,6 +839,21 @@ GetOptions getGetOptions() {
     }
 
     /**
+     * Return the {@link MessageFormatFlags} to associate with the first getBlob chunk operation.
+     * @return {@link MessageFormatFlags#Blob} for {@link GetBlobOptions.OperationType#Data}, or {@link MessageFormatFlags#All} by
+     *         default.
+     */
+    @Override
+    MessageFormatFlags getOperationFlag() {
+      switch (options.getOperationType()) {
```

Comment:
considering how there are only two cases, I'd choose `if-else` (or the ternary operator here) over `switch-case`.